README.md

```diff
@@ -346,6 +347,21 @@ python server.py
 mcp run server.py
 ```
 
+### Mounting to an Existing ASGI Server
+
+You can mount the SSE server to an existing ASGI server using the `sse_app` method. This allows you to integrate the SSE server with other ASGI applications.
+
+```python
+from fastapi import FastAPI
+from mcp.server.fastmcp import FastMCP
+
+app = FastAPI()
```

Comment:
Can you use starlette and link to the starlette docs about mounting instead?

Reply:
Sure. It seems more natural to use the Starlette example.